### DISCUSSION TITLE
eBook 2 - two questions

### DISCUSSION THREAD

**Discussion by robs-fotoalbum** [typemill/typemill#434] [Q&A]
I just installed the eBook 2 plugin. As it was not seen by Typemill immediately, I renamend the folder from the zip _ebook 2_ to _ebook_ - then it was visible in the plugin section.

I started to experimented with both layouts, both are working, but _report_ doesn't render footnotes as _manual_ does. But the rendered footnotes in _manual_ don't render hyperlinks.

**Comment by trendschau** [comment]
Thank you, I will check it for the next version of the ebook-plugin! folder should be named ebook instead of ebook, I will check if there is a mistake in the zip. The manual should work with Hyperlinks, maybe that is broken, I will check it too.

**Comment by trendschau** [comment]
Updated the ebook-plugin and fixed some errors. Added footnotes to reports, the hyperlinks should be there but they are not styled. I will improve that with the next release of the ebook plugin.
